subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2227,25 +2338,16 @@ static int handle_request(struct net_app_ctx *app_ctx,
 	return r;
 }
 
-int lwm2m_udp_sendto(struct net_app_ctx *app_ctx, struct net_pkt *pkt)
-{
-	return net_app_send_pkt(app_ctx, pkt, &app_ctx->default_ctx->remote,
-				NET_SOCKADDR_MAX_SIZE, K_NO_WAIT, NULL);
-}
-
 void lwm2m_udp_receive(struct lwm2m_ctx *client_ctx, struct net_pkt *pkt,
 		       bool handle_separate_response,
-		       int (*udp_request_handler)(struct net_app_ctx *app_ctx,
-						  struct zoap_packet *,
-						  struct zoap_packet *))
+		       udp_request_handler_cb_t udp_request_handler)
 {
 	struct net_udp_hdr hdr, *udp_hdr;
 	struct zoap_pending *pending;
 	struct zoap_reply *reply;
 	struct zoap_packet response;
 	struct sockaddr from_addr;
-	struct zoap_packet response2;
-	struct net_pkt *pkt2;
+	struct lwm2m_message *msg = NULL;
```

Comment:
in net subsystem/libs, we try to enforce declaration with assignment lines put always first. so this line would need to be below {